src/rules/at-root-no-redundant/README.md

```diff
@@ -0,0 +1,67 @@
+# at-root-no-redundant
+
+Ban unnecessary `@at-root` rule.
```

Comment:
There are some small things to point out related to how the README and message should be formatted. 

Please check Stylelint's developer guide for more info:
https://stylelint.io/developer-guide/rules#write-the-readme

```md
The single-line description is in the form of:

    - "Disallow ..." for no rules
    - "Limit ..." for max rules
    - "Require ..." for rules that accept "always" and "never" options
    - "Specify ..." for everything else
```